==== //sandbox/gboyce/sieve/python-managesieve-0.3/managesieve.py#9 - /Users/gboyce/perforce/sandbox/sieve/python-managesieve-0.3/managesieve.py ====
--- managesieve.py      Sat May 24 13:45:40 2003
+++ managesieve.py.new  Mon Aug 25 13:58:52 2008
@@ -135,6 +135,13 @@
 
 
     def _parse_capabilities(self, lines):
+        # XXX darky: seeing the STARTTLS cap makes managesieve barf
+        # because 'data' (in the below unpack) has nothing to unpack.
+        # Fix up lines for this.
+        for pair in lines:
+            if len(pair) == 1:
+                pair.append("")
+            assert len(pair) == 2
         for typ, data in lines:
             if __debug__:
                 if self.debug >= 3:
